security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -17,6 +17,72 @@ To access the latest release, see xref:../../security/compliance_operator/co-man
+// For more information, see (link:https://issues.redhat.com/browse/CMP-3596[CMP-3596]).
+
+* In previous releases of Compliance Operator, the rule `audit-log-forwarding-uses-tls` would fail even if tls was enabled for `clusterloggerforwarder`. With this release, `audit-log-forwarding-uses-tls` shows `PASS` when tls is enabled for `clusterloggerforwarder`. For more information, see (link:https://issues.redhat.com/browse/CMP-3597[CMP-3597]).
+
+* In prior versions of Compliance Operator, for the rule `rhcos4-service-systemd-coredump-disabled`, no remediation was generated after scanning the cluster. In this release, remediation is provided for `rhcos4-service-systemd-coredump-disabled`. 
+// For more information, see (link:https://issues.redhat.com/browse/CMP-3599[CMP-3599]).
+
+* To periodically check for updates to containers, administrators use the configuration of `imagestream.spec.tags.importPolicy.scheduled`. This is useful when working with an external container image registry, to periodically re-import an image, for example, to get latest security updates. In prior versions of Compliance Operator, the rule to check the setting of `imagestream.spec.tags.importPolicy.scheduled` would return `FAIL` regardless. With this release, upon correct application of remediation, Compliance Operator no longer reports `FAIL`. 
+// For more information, see (link:https://issues.redhat.com/browse/CMP-3601[CMP-3601]).
+
+* In prior releases of Compliance Operator, when remediations were applied for updating TLS ciphers for the `kubeapi` server, the Compliance Operator did not have needed permissions to make the change. With this release, Compliance Operator is able to correctly remediate ciphers for the `kubeapi` server. For more information, see (link:https://issues.redhat.com/browse/CMP-3606[CMP-3606]).
+
+* In prior versions of Compliance Operator, the creation of Compliance Scans would cause problems with the checking of deprecated profiles. With this release, Compliance Operator checks for deprecated profiles are no longer failing. For more information, see (link:https://issues.redhat.com/browse/CMP-3613[CMP-3613]).
+
+* In prior releases of Compliance Operator, when running `ComplianceRemediations`, sshd access would be killed with a `signal 11` whenever the file `/etc/ssh/ssh_config` contained the text: "Include /etc/ssh/sshd_config.d/*.conf". With this release, Compliance Operator provides correct remediation and does not kill sshd. 
+// For more information, see (link:https://issues.redhat.com/browse/CMP-3624[CMP-3624]). 
+ 
+* Previously, Compliance Operator scans triggered the `ocp4-routes-protected-by-tls1` rule for passthrough routes which should not be marked as `insecure` in the remediation results. With this release, passthrough routes are correctly scanned and remediated. 
```

Comment:
Two minor issue:
1. typo error in the rule name.
2.  "correctly scanned and remediated" - This is slightly misleading. The PR actually excludes passthrough routes from the rule entirely, rather than "correctly remediating" them.
How about
"Previously, Compliance Operator scans incorrectly flagged passthrough routes as NON-COMPLIANT with the `ocp4-routes-protected-by-tls` rule. With this release, passthrough routes are properly excluded from this rule since they delegate TLS termination to the backend application."